recipes/current_events_magnifier/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "current_events_magnifier" %}
+{% set version = "0.0.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2719b11ab8046909e6b34907f0a0c3368b22c45df5f4769746dd3a47ebef73b6
+
+build:
+  number: 0
+  noarch: generic
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - h5py ==3.8.0
+    - numpy >=1.23.0
+    - pandas >=1.1.5
+    - pip
+    - plotnine >=0.8.0
+    - pyslow5 >=1.0.0
+    - pysam >=0.21.0
+    - python
+    - tqdm >=4.62.0
+  run:
+    - h5py ==3.8.0
+    - numpy >=1.23.0
+    - pandas >=1.1.5
+    - plotnine >=0.8.0
+    - pyslow5 >=1.0.0
+    - pysam >=0.21.0
+    - python
+    - tqdm >=4.62.0
+
+test:
+  imports:
+    - current_events_magnifier
+
+about:
+  home: "https://github.com/lrslab/current_events_magnifier"
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
This here is missing and please add the doc_url or clean up the recipe a bit. Thanks.